x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/DownsampleActionIT.java

```diff
@@ -396,7 +411,12 @@ public void testRollupNonTSIndex() throws Exception {
         // Create the ILM policy
         String phaseName = randomFrom("warm", "cold");
         DateHistogramInterval fixedInterval = ConfigTestHelpers.randomInterval();
-        createNewSingletonPolicy(client(), policy, phaseName, new DownsampleAction(fixedInterval, DownsampleAction.DEFAULT_WAIT_TIMEOUT));
+        createNewSingletonPolicy(
+            client(),
+            policy,
+            phaseName,
+            new DownsampleAction(fixedInterval, DownsampleAction.DEFAULT_WAIT_TIMEOUT, randomBoolean())
```

Comment:
I think it would be valuable to have one test that explicitly enables force merge and verifies that the downsampled index has 1 segment.

Reply:
> but I do not know how to ensure that it will have more segments originally

This is indeed not possible. I'd say that if it generally has more than one segment originally and it only rarely doesn't have more (which is why we can't ensure it), then that is still valuable as a test. What do you think?

Reply:
I will check if there is any way to configure the test in a way to force the downsampled index to have more segments. But if there is no way to do that, I think it's not worth it.

Reply:
I found https://www.elastic.co/docs/reference/elasticsearch/index-settings/translog, there are a few things to explore here, if small enough values are allowed and if the downsample action allows me to set this values.